frontend/public/src/components/CourseProductDetailEnroll.js

```diff
@@ -199,28 +330,24 @@ export class CourseProductDetailEnroll extends React.Component<
           id={`upgrade-enrollment-dialog`}
           className="upgrade-enrollment-modal"
           isOpen={upgradeEnrollmentDialogVisibility}
-          toggle={() => this.toggleUpgradeDialogVisibility()}
+          toggle={() => this.cancelEnrollment()}
           centered
         >
-          <ModalHeader toggle={() => this.toggleUpgradeDialogVisibility()}>
+          <ModalHeader toggle={() => this.cancelEnrollment()}>
             {run.title}
           </ModalHeader>
           <ModalBody>
-            <div className="row">
-              <div className="col-12">
-                <p>
-                  Thank you for choosing an MITx Online course. By paying for
-                  this course, you're joining the most engaged and motivated
-                  learners on your path to a certificate from MITx.
-                </p>
+            {courseRuns && courseRuns.length > 1 ? (
+              <div className="row date-selector-button-bar">
+                <div className="col-12">
+                  <div>{this.renderRunSelectorButtons(run)}</div>
+                </div>
               </div>
-            </div>
+            ) : null}
 
-            <div className="row">
-              <div className="col-12">
-                <p className="acheiving-text">
-                  Acheiving a certificate has its advantages:
-                </p>
```

Comment:
We need a clearer call to action here. 

"Do you want you want to earn a certificate?"

Reply:
There's a typo in ^that, just in case you were going to copy/paste.